app/src/main/res/values/strings.xml

```diff
@@ -643,6 +643,7 @@
 
     <string name="pref_title_show_notifications_filter">Show Notifications filter</string>
     <string name="pref_title_enable_swipe_for_tabs">Enable swipe gesture to switch between tabs</string>
+    <string name="pref_title_show_stat_inline">Show toot statistics</string>
```

Comment:
You can use "post" instead of "toot".

Maybe word it "Show post statistics in timeline"?